src/api/java/com/minecolonies/api/blocks/AbstractBlockHut.java

```diff
@@ -77,7 +77,7 @@
     private static final VoxelShape SHAPE = VoxelShapes.create(0.1, 0.1, 0.1, 0.9, 0.9, 0.9);
 
     /**
-     * The hut's lower-case building-registry-compatible name.
+     * The hut's lower-case building-registry-compatible name.s
```

Comment:
is this change intentional?

Reply:
No. Also, damn you're fast.

Reply:
heh. fastest triage in the west, and all that

Reply:
In fact, I didn't even touch that file, my fork just has weird crap happening. I no longer have direct access to the repo, so that's a thing.